utils/url.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Converts an object into a URI query string.
+ * Supports arrays and nested objects.
+ */
+// eslint-disable-next-line
+export function toQueryString(object: Record<string, any>, prefix = ''): string {
+  if (object === undefined || object === null) {
+    return '';
+  }
+
+  const query = Object.keys(object).map((key) => {
+    const value = object[key];
+
+    if (object.constructor === Array) {
+      key = prefix;
+    } else if (object.constructor === Object) {
+      key = prefix ? `${prefix}[${key}]` : key;
+    }
+
+    if (typeof value === 'object') {
+      return toQueryString(value, key);
+    } else if (value === undefined || value === null) {
+      return '';
+    } else {
+      return `${key}=${encodeURIComponent(value)}`;
+    }
+  });
+
+  return query.filter((str) => str.length > 0).join('&');
+}
```

Comment:
Hey Siva, is there a reason we can't use the axios way of handling query parameters?

```
await api.get('path', { params }).then(...);
```

params supports all the normal objects expected in a GET query, and I feel its safer to trust the axios implementation.